inspirehep/records/api/literature.py

```diff
@@ -77,18 +77,14 @@ def earliest_date(self):
         dates = [
             str(el)
             for el in chain.from_iterable(
-                force_list(self.get_value(path)) for path in date_paths
+                [force_list(self.get_value(path)) for path in date_paths]
             )
         ]
-        proper_dates = []
-        for date in dates:
-            try:
-                proper_dates.append(dateutil.parser.parse(date))
-            except ValueError:
-                pass
-        if proper_dates:
-            date = earliest_date(dates, full_date=True)
```

Comment:
the `full_date` is needed for the citation count table, I don't think it matters for ES.